os/board/rtl8721csm/src/component/common/network/ssl/ssl_ram_map/rom/rom_ssl_ram_map.h

```diff
@@ -0,0 +1,108 @@
+#ifndef ROM_SSL_RAM_MAP_H
```

Comment:
Please insert copyright.

Reply:
Inserted Realtek copyright.

Reply:
I think the source file can't be uploaded to public git with this copy right. the copy right should be APACHE like license or samsung license.
please check the copy right